types/options.go

```diff
@@ -13,11 +13,12 @@ import (
 	"github.com/containers/storage/drivers/overlay"
 	cfg "github.com/containers/storage/pkg/config"
 	"github.com/containers/storage/pkg/idtools"
+	"github.com/containers/storage/pkg/unshare"
```

Comment:
This is a large dependency to pull in for a package named "types".

Reply:
should I just pass a rootless bool to the function? @rhatdan WDYT?

Reply:
Yes